tox.ini

```diff
@@ -30,7 +31,9 @@ commands =
 [testenv:coverage]
 deps =
     pytest
+    pytest-ordering
     pytest-cov
+
```

Comment:
Can someone who knows tox tell me the right way to do this?

Seems like we shouldn't have to add dependencies in more than one place, but I couldn't get the tests passing without this.  Maybe the way I created the coverage command is part of the problem, but even so we're already specifying test deps in our `setup.cfg` — can we use that here somehow?

Reply:
Haven't used `tox` except here unfortunately. Looks like these are some instructions on how to configure `tox` to use `setup.cfg` settings: https://tox.wiki/en/latest/config.html#configuration-discovery. Found that from https://stackoverflow.com/questions/67098213/is-there-a-reason-to-have-both-tox-ini-and-setup-cfg-in-a-python-project

Reply:
Thanks @ColeDCrawford — I saw in passing that tox could be configured in `setup.cfg`, that sounds appealing to me but not sure the implications